### ISSUE TITLE
Enquiry of success

### ISSUE THREAD

**Issue by Hobo17** [rainbowdashobard/docker-steamcmd-pixark#1]
Hey, I was trialling your docker setup here and was wondering if you've been able to get this to work? I've tried Exposing extra ports, opening an excess more, building the Dockerfile myself, making sure the ulimit is set correctly and even mounting a volume for the server so testing was faster. I've probably over-tested this but I can't seem to get it to work, so I thought I'd ask if you had success or not.

**Comment by rainbowdashobard** [comment]
Unfortunately no success on my end either. I forked this off of an existing SteamCMD base, so I know that -should- work. I'm not certain of Pixark has any LAN-Only server system so I can only assume its a port forwarding problem.

**Comment by Hobo17** [comment]
Yeah, the furtherest I've gotten is a bunchload of 'LogAnimation:Warning: FBoneReference::Initialize BoneIndex for Bone 'Root' does not exist in Skeleton ''SM_cw_animal_hippocampus_skeleton''s and 'biome not exists'. Which according to some people was due to CubePort not being forwarded, but I'm rather certain my ports were set up correctly. The server works on my main Windows PC but it doesn't use steamcmd or anything I suppose, just doesn't let people join and play. (Sorry for the late one, I crashed after it failed too many times and it was 2am. Haha)

**Comment by moontooth719096** [comment]
I think CubePort is not exposed

![image](https://user-images.githubusercontent.com/31216214/175794733-fa3660c4-c293-4619-b033-63e41095f4dd.png)

![image](https://user-images.githubusercontent.com/31216214/175794738-788fc1df-9d13-45cd-9bcb-04b51d2ffb96.png)
